allium/templates/network-health-dashboard.html

```
@@ -377,21 +377,83 @@ <h4><a href="#operator-participation" class="al-link-plain">👥 Operator Partic
                         <div class="al-text-small-muted metric-subtext">{{ "%.1f%%"|format(relays.json.network_health.relays_without_aroi_percentage) }} of all relays</div>
                     </div>
                 </div>
-                <!-- DNS-RSA and URI-RSA Validation Rates -->
+                <!-- B2.1: v2 (RSA-fingerprint) proof types -->
                 <div class="metric-grid" style="grid-template-columns: repeat(2, 1fr);">
-                    <div class="metric-item" title="DNS-RSA validation: {{ "{:,}".format(relays.json.network_health.dns_rsa_valid) }}/{{ "{:,}".format(relays.json.network_health.dns_rsa_total) }} relays validated. DNS-RSA uses DNS TXT records for domain verification.{% if relays.json.network_health.dns_error_top5 %}
+                    <div class="metric-item" title="DNS-RSA (v2): {{ "{:,}".format(relays.json.network_health.dns_rsa_valid) }}/{{ "{:,}".format(relays.json.network_health.dns_rsa_total) }} relays validated. CIISS v2 — DNS TXT record at <fingerprint>.<domain>.{% if relays.json.network_health.dns_error_top5 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)